Makefile

```diff
@@ -17,9 +20,9 @@ cover:  ## Run test coverage suite
 	@go tool cover --html=cov.out
 
 build:  ## Builds the CLI
-	@go build \
-	-ldflags "-w -s -X ${PACKAGE}/cmd.version=${VERSION} -X ${PACKAGE}/cmd.commit=${GIT} -X ${PACKAGE}/cmd.date=${DATE}" \
-	-a -tags netgo -o execs/${NAME} main.go
+	@${GO_BUILD} ${GO_FLAGS} \
```

Comment:
I am ok with adding  build flags here but overriding the build command seems a tad overkill

Reply:
Ok, I can change